test/test_examples_3d.jl

```diff
@@ -111,42 +124,119 @@ const EXAMPLES_DIR = joinpath(pathof(Trixi) |> dirname |> dirname, "examples", "
 
 
   @testset "taal-confirmed elixir_euler_taylor_green_vortex.jl" begin
-  test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_euler_taylor_green_vortex.jl"),
+    test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_euler_taylor_green_vortex.jl"),
       l2   = [0.0003494971047256544, 0.03133386380969968, 0.031333863809699644, 0.04378595081016185, 0.015796569210801217],
       linf = [0.0013934701399120897, 0.07284947983025436, 0.07284947983025408, 0.12803234075782724, 0.07624639122292365],
       tspan = (0.0, 0.5))
   end
 
   @testset "taal-confirmed elixir_euler_shockcapturing.jl" begin
-  test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_euler_shockcapturing.jl"),
+    test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_euler_shockcapturing.jl"),
       l2   = [0.025558219399128387, 0.01612806446620796, 0.016128064466207948, 0.016120400619198158, 0.09208276987000782],
       linf = [0.3950327737713353, 0.26324766244272796, 0.2632476624427279, 0.2634129727753079, 1.371321006006725])
   end
 
+  @testset "taal-confirmed elixir_euler_shockcapturing.jl with initial_condition_sedov_blast_wave" begin
+    # OBS! This setup does not run longer but crashes (also the parameters do not make sense) -> only for testing the IC!
+    test_trixi_run(joinpath(EXAMPLES_DIR, "elixir_euler_shockcapturing.jl"),
+      l2   = [0.03627060784392582, 0.05178777376859809, 0.05178777376859804, 0.05178777376859802, 0.23043996953698023],
+      linf = [0.9307998233177583, 1.4326649193439467, 1.4326649193439467, 1.4326649193439467, 12.80585041235138],
+      initial_condition=Trixi.initial_condition_sedov_blast_wave, cfl=0.25, shock_alpha_max=1.0, t_end=0.1)
```

Comment:
This IC should be exported (it is on `dev`).

Reply:
Fixed in a2bedcb.